test/test_transforms_v2_refactored.py

```diff
@@ -2126,16 +2127,10 @@ class TestGetKernel:
         datapoints.Video: F.resize_video,
     }
 
-    def test_unsupported_types(self):
-        class MyTensor(torch.Tensor):
-            pass
```

Comment:
Why was this one removed?

Reply:
Because this will no longer fail. Subclasses of `PIL.Image.Image` and `torch.Tensor` are now allowed by `_get_kernel`. As explained in https://github.com/pytorch/vision/pull/7835#issue-1852715293, this has no effect on the user, since they cannot register anything like that.